src/random.js

```diff
@@ -43,6 +44,14 @@ export const randomString = ({chance = chanceInstance, inputValue = VALUE_DEFAUL
   return chance.word({length: inputValue})
 }
 
+export const randomHash = ({chance = chanceInstance, inputValue = VALUE_DEFAULT_HASH_LENGTH}) => {
```

Comment:
What do you think of having some unit tests for this function?

Reply:
👍 

I implemented it quickly so I could get it running locally, and wasn't sure I had added it in all the correct places. Will add tests.